lib/helper.js

```diff
@@ -145,7 +145,7 @@ function jsxTranspile(instrument) {
     var fs = require('fs');
     var babel = require('babel');
 
-    if (instrument !== undefined && instrument && instrument === 'true') {
+    if (instrument && instrument === 'true') {
```

Comment:
but the way it is it would fail for `instrument === true`... why do you want to pass the instrument as a string instead of a boolean?